templates/placementapi/config/logging.conf

```diff
@@ -26,6 +26,12 @@ level=WARNING
 formatter=normal
 args=(sys.stdout,)
 
+[handler_fileHandler]
+class=FileHandler
+level=WARNING
```

Comment:
what would be a way to set DEBUG?

Reply:
this is i guess another pices of tech debt
we fully removed the longing conf form nova we should do that here also.

the correct way to enable debug logging is via the placement.conf

the problem with the placement operator is we still have not rewitten its config generation.
that was a task we had lining up for some for this cycle but then eded up not working on the placement operator at all.

Reply:
lets just set this to debug level for now in the logger config and we can then control debug level via the customerServiceConfig

we should be defaulting to debug.

Reply:
done and also set debug = true in placement.conf